.gitignore

```diff
@@ -0,0 +1 @@
+venv/
```

Comment:
can we not?

please don't add to `.gitignore` files/directories that are specific for your system/case, if we all add their dev directory to `.gitignore` that might lead to unforeseen consequences and confusion down the road :) 

you can have this in your own `.gitignore` without adding it to commits